src/core/generators/verbsOptions.ts

```diff
@@ -206,7 +206,12 @@ export const generateVerbsOptions = ({
           context,
         });
 
-        acc.push(verbOptions);
+        // GitHub #564 check if we want to exclude deprecated operations
+        const includeOperation = !operation.deprecated || output.override.useDeprecatedOperations === true;
+
+        if (includeOperation) {
+          acc.push(verbOptions);
+        }
```

Comment:
I apologize if this is not the correct place for this check and exclusion but it felt like the only place that made sense but feel free to make a better suggestion!

Reply:
I have more work to do here. Wasn't as simple as I hoped. Was hoping to just pull the whole operation out of the list. Before it's processed.